pizza-time/src/components/card/card.js

```diff
@@ -120,8 +125,12 @@ class Card extends React.Component {
                   return [attende.first_name, "  ", attende.last_name, ","," "]
```

Comment:
again here you can use the template literal syntax `${attendee.first_name} ${attendee.last_name} `